### PR TITLE
Test how we can use CloudFlare Pages's redirects

### PR DESCRIPTION
**Pull request by zupo**
This commit needs to be merged to master to verify Cloudflare correctly picks it up. I will revert it after this is verified.

Needed for https://github.com/NixOS/nix.dev/pull/353

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,6 +51,10 @@ clean:
 .PHONY: html
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+
+	# support for Cloudflare Pages' redirects
+	@cp _redirects build/html/_redirects
```
Comment (lucperkins) [inline Makefile:56]: Instead of this, you can add the file directly to `source/_static` and Sphinx automatically adds it to the build output

**Review by lucperkins — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (zupo) [reply Makefile:56]: Oh, sweet, lemme try that.

Comment (zupo) [reply Makefile:56]: Sadly, does not seem to work. At least on my local the file gets put into `_static/` output folder, not into root. 

Screenshot 2022-10-22 at 15 49 20

Comment (lucperkins) [reply Makefile:56]: Never mind

**Comment by zupo** [comment]
https://developers.cloudflare.com/pages/platform/redirects/ recommends changing the build command to something like `jekyll build && cp _redirects _site/_redirects` and we might have to do that. 

I asked @domenkozar to give me access to Cloudflare Pages.

**Comment by zupo** [comment]
Turns out, the Cloudflare Pages build failure was random, re-running it fixed it. 

Also, no need to merge this, Cloudflare Pages preview is enough. I've verified the redirects works on a preview page.

```
➜  nix.dev git:(master) ✗ curl --head https://f80dea2a.nix-dot-dev.pages.dev/foo
HTTP/2 302 
```

**Pull request closed without merging**